website/docs/guides/manual-install-qs.md

```diff
@@ -36,7 +36,7 @@ The following steps use [GitHub](https://github.com/) as the Git provider for th
 2. Select **Public** so the repository can be shared with others. You can always make it private later.
 3. Leave the default values for all other settings.
 4. Click **Create repository**.
-5. Save the commands from "…or create a new repository on the command line" to use later in [Commit your changes](#commit-your-changes).
+5. Save the commands from "…or create a new repository on the command line" to use later in [Commit your changes](https://docs.getdbt.com/guides/manual-install?step=6).
```

Comment:
Hiya @mirnawong1 

I'm probably reading this very wrong but thought I'd ask just in case. Do you mean the user needs to save the commands from another location (ie is there a word / phrase missing) or will they be prompted to save the commands from this section > "or create a new repository on the command line"?

Kind Regards
Natalie